modules/@apostrophecms/image/index.js

```diff
@@ -19,11 +19,6 @@ module.exports = {
     sort: { createdAt: -1 },
     insertViaUpload: true,
     searchable: false,
-    browser: {
```

Comment:
Something like this is desirable, but options don't merge (they never have), a new setting for a top level option blows out an inherited one. So for now this pattern doesn't work, we must extend the getBrowserData method. We could discuss a feature to provide option merging at some point. It's never as simple as it seems...